includes/polling/ports.inc.php

```diff
@@ -641,6 +631,20 @@
 
             $oid_prev = $oid.'_prev';
             if (isset($port[$oid])) {
+                if ($device['uptime'] < $config['rrd']['step']) {
```

Comment:
This block attempts to address counter rollover for ports in 32 bit mode.

If the device hasn't recently rebooted and the current Octet value is < then the previous something has gone wrong. So we subtract the previous Octet value from the max possible for 64bit and then add the current one onto that value.